pages/documentation/developer/repositories/index.md

```diff
@@ -46,6 +46,12 @@ This [repository](https://github.com/powsybl/powsybl-single-line-diagram) provid
 **Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst)
 **Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst)
 
+### [powsybl-metrix](powsybl-metrix.md) [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-metrix.svg)](https://github.com/powsybl/powsybl-metrix/releases/)
+This [repository](https://github.com/powsybl/powsybl-metrix) provides modules to run optimal power load flow on multi case configured network.
+
+**Reviewers:** [pl-buiquang](https://github.com/pl-buiquang), [geofjamg](https://github.com/geofjamg)
+**Committers:** [pl-buiquang](https://github.com/pl-buiquang), [mathbagu](https://github.com/mathbagu), [MioRtia](https://github.com/MioRtia)
```

Comment:
Can you change for @bongrainmat for the C++ part and @berthaultval and you for the Java part for committers? For the releases, it will be @MioRtia in back-up but it could be nice that someone from your team can manage it.